client/lib/plans/features-list.js

```diff
@@ -477,7 +483,11 @@ export const FEATURES_LIST = {
 	},
 	[ constants.FEATURE_SIMPLE_PAYMENTS ]: {
 		getSlug: () => constants.FEATURE_SIMPLE_PAYMENTS,
-		getTitle: () => i18n.translate( 'Simple Payments' ),
+		getTitle: () => {
+			isEnabled( 'earn/rename-payment-blocks' )
+				? i18n.translate( 'Pay with PayPal' )
+				: i18n.translate( 'Simple Payments' );
```

Comment:
We need to return this statement. (Notice the brace change).

<img width="606" alt="Screen Shot 2020-06-22 at 4 42 04 PM" src="https://user-images.githubusercontent.com/1270189/85345607-b48e8a80-b4a7-11ea-8ab9-f296779bc242.png">